src/constants/logoMap.json

```diff
@@ -1,5 +1,7 @@
 {
   "django": "django-logo.png",
+  "html": "html-logo.png",
+  "select-2": "select2-logo.png",
```

Comment:
@libinmv could you please add another key `select2` with the same file name

Reply:
ok